cmd/clusterctl/client/cluster/ownergraph.go

```diff
@@ -79,3 +96,12 @@ func GetOwnerGraph(namespace, kubeconfigPath string) (OwnerGraph, error) {
 	}
 	return owners, nil
 }
+
+// isClusterSecret checks whether a Secret is related to a CAPI Cluster by checking if the secret type is ClusterSecretType.
+func isClusterSecret(ref corev1.ObjectReference, c client.Client) (bool, error) {
```

Comment:
I've tested this locally with Kubernetes 1.20.15 and it works. I think this approach makes sense for what we're currently testing, but we might want to be careful of expectations if providers pick up this test.